subprojects/gradle/impact/build.gradle.kts

```diff
@@ -28,6 +31,13 @@ dependencies {
     gradleTestImplementation(testFixtures(project(":common:statsd")))
     gradleTestImplementation(project(":gradle:test-project"))
     gradleTestImplementation(project(":gradle:impact-shared-test-fixtures"))
+
+    // workaround for https://github.com/gradle/gradle/issues/16774
+    gradleTestRuntimeOnly(
+        files(
+            serviceOf<ModuleRegistry>().getModule("gradle-tooling-api-builders").classpath.asFiles.first()
+        )
+    )
```

Comment:
Does it worth it to hide it inside conventions plugins and apply it everywhere?

Reply:
i thought about special convention for "InMemoryProjects", which will extend gradle test convention
but for temporary workaround? probably not worth

Reply:
How about `gradle-testing`?

Reply:
only 3 modules uses this hack, so maybe it's an overkill

Reply:
As for me, the second repetition is enough. Especially when it's hard to diagnose the root cause.
up to you

Reply:
https://github.com/avito-tech/avito-android/pull/1095